app/models/concerns/language_and_schedule_references.rb

```diff
@@ -5,6 +5,20 @@ module LanguageAndScheduleReferences
     has_many :language_skills, as: :languageable, dependent: :destroy
     accepts_nested_attributes_for :language_skills, allow_destroy: true
 
+    def first_language
+      return [] if language_skills.blank? || language_skills.native.blank?
+      return language_skills.native.german if language_skills.native.german.present?
+      return language_skills.native.limit(1) if language_skills.native.size.positive?
+    end
```

Comment:
There are multiple solutions for the `nil` case:

1. The view checks the return value: `volunteer.first_language.language_name if volunteer.first_language`
1. Use rails' `try`: `volunteer.first_language.try(:language_name)
1. Use ruby's safe operator: `volunteer.first_language&.language_name.
1. Return a valid (but empty) object in any case: `language_skills.native.german.first || language_skills.native.first || LanguageSkill.new`